pydy/viz/static/index_ipython.html

```diff
@@ -10,7 +10,8 @@
     <link type="text/css" rel="stylesheet" href="static/css/main.css">
     <link type="text/css" rel="stylesheet" href="static/css/codemirror/blackboard.css">
     <link type="text/css" rel="stylesheet" href="static/css/slider.css">
-    <link type="text/css" rel="stylesheet" href="static/css/bootstrap.min.css">
+    <!-- commented import of bootstrap until css leak is fixed in Ipython widgets-->
+    <!--<link type="text/css" rel="stylesheet" href="static/css/bootstrap.min.css">-->
```

Comment:
I think to fix the leak issue they will have to fix it both ways.ie. widget's css should not be applied to the notebook's css  as well as notebook's css should not be applied to widget.